Vanilla binutils need the below to be able to build using GCC with Google patches, since
gas sometimes gets passed --save-temps in that environment.

diff -u -N -r binutils-2.24.orig/gas/as.c binutils-2.24/gas/as.c
--- binutils-2.24.orig/gas/as.c	2013-11-04 07:33:37.000000000 -0800
+++ binutils-2.24/gas/as.c	2014-01-13 14:11:28.677467167 -0800
@@ -328,6 +328,8 @@
                           prefer smaller memory use at the cost of longer\n\
                           assembly times\n"));
   fprintf (stream, _("\
+  --save-temps            ignored\n"));
+  fprintf (stream, _("\
   --statistics            print various measured statistics from execution\n"));
   fprintf (stream, _("\
   --strip-local-absolute  strip local absolute symbols\n"));
@@ -455,7 +457,8 @@
       OPTION_REDUCE_MEMORY_OVERHEADS,
       OPTION_WARN_FATAL,
       OPTION_COMPRESS_DEBUG,
-      OPTION_NOCOMPRESS_DEBUG
+      OPTION_NOCOMPRESS_DEBUG,
+      OPTION_SAVE_TEMPS
     /* When you add options here, check that they do
        not collide with OPTION_MD_BASE.  See as.h.  */
     };
@@ -518,6 +521,7 @@
     ,{"nocpp", no_argument, NULL, OPTION_NOCPP}
     ,{"no-warn", no_argument, NULL, 'W'}
     ,{"reduce-memory-overheads", no_argument, NULL, OPTION_REDUCE_MEMORY_OVERHEADS}
+    ,{"save-temps", no_argument, NULL, OPTION_SAVE_TEMPS}
     ,{"statistics", no_argument, NULL, OPTION_STATISTICS}
     ,{"strip-local-absolute", no_argument, NULL, OPTION_STRIP_LOCAL_ABSOLUTE}
     ,{"version", no_argument, NULL, OPTION_VERSION}
@@ -611,6 +615,9 @@
 	case OPTION_NOCPP:
 	  break;
 
+	case OPTION_SAVE_TEMPS:
+	  break;
+
 	case OPTION_STATISTICS:
 	  flag_print_statistics = 1;
 	  break;
